test/lib/assert-equal-dom.js

```diff
@@ -31,7 +32,12 @@ function areEqual(a, b) {
                 return equalStyle(a[key], b[key])
             }
             if (typeof a === "object" || typeof a === "function") {
-                if (!areEqual(a[key], b[key])) {
+                if (cache.indexOf(a) !== -1) {
```

Comment:
FWIW not even the IE team wants to support IE8 anymore. https://github.com/emberjs/rfcs/pull/45#issuecomment-89627765